pkg/frontend/validate.go

```diff
@@ -118,3 +118,22 @@ func validateAdminVMName(vmName string) error {
 
 	return nil
 }
+
+func validateAdminKubernetesPodLogs(method, name, namespace, containerName string) error {
+	if method != http.MethodGet {
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, "", "The provided HTTP method '%s' is invalid.", method)
+	}
+
+	if name == "" {
```

Comment:
Use `rxKubernetesString` regex to validate this.

Reply:
done